runtime/doc/develop.txt

```diff
@@ -311,27 +516,29 @@ get_env_value()	Linux system function
 
 VARIOUS							*style-various*
 
-Typedef'ed names should end in "_T": >
-    typedef int some_T;
-Define'ed names should be uppercase: >
+Define'ed names should be uppercase:
```

Comment:
[Fowler](https://en.wikipedia.org/wiki/A_Dictionary_of_Modern_English_Usage) suggests ‘long-pedigree'd’.

So, ‘Typedef'd’ and ‘Define'd’.